src/node_crypto.cc

```diff
@@ -943,40 +949,48 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
 }
 
 
-void SecureContext::SetCiphers(const FunctionCallbackInfo<Value>& args) {
+#if 1
+#define fprintf(...)
+#endif
```

Comment:
Side note: `Debug()` from `debug_utils.h` might have something – it works out-of-the-box for async wraps, since they already have a limited set of names, but it should be easy enough to add other categories?